### DISCUSSION TITLE
Spam folder icon is a bug

### DISCUSSION THREAD

**Discussion by renevandenberg** [tutao/tutanota#3836] [Ideas]
The spam folder icon is currently a bug symbol. Spam is not a bug as it is not something causing errors but something that is simply annoying. 

Suggestions: exclamation mark, cross sign, not allowed sign 🚫⛔, mosquito 🦟, canned meat, disappointed face 😞

**Comment by mrx23dot** [comment]
Why not some pork symbol?
